src/components/Register/RegisterForm/index.tsx

```diff
@@ -0,0 +1,187 @@
+        {error && <Error error={error} />}
+        {loading ? (
+          <Loading />
+        ) : (
+          <form onSubmit={handleSubmit}>
+            <div className={`form-group ${styles.formGroupCustom}`}>
+              <label
+                className={
+                  firstName
+                    ? `${styles.floatLabel}`
+                    : `${styles.noLabel}`
+                }
+              >
+                {' '}
+                First Name:
+              </label>
+              <input
+                type="text"
+                value={firstName}
+                placeholder="First Name"
+                onChange={e => setFirstName(e.target.value)}
+                className="form-control"
+                required
+                autoFocus
+              />
+            </div>
+            <div className={`form-group ${styles.formGroupCustom}`}>
+              <label
+                className={
+                  lastName
+                    ? `${styles.floatLabel}`
+                    : `${styles.noLabel}`
+                }
+              >
+                {' '}
+                Last Name:
+              </label>
+              <input
+                type="text"
+                value={lastName}
+                placeholder="Last Name"
+                onChange={e => setLastName(e.target.value)}
+                className="form-control"
+                required
+              />
+            </div>
+            <div className={`form-group ${styles.formGroupCustom}`}>
+              <label
+                className={
+                  email ? `${styles.floatLabel}` : `${styles.noLabel}`
+                }
+              >
+                Email
+              </label>
+              <input
+                type="email"
+                value={email}
+                placeholder="Email"
+                onChange={e => setEmail(e.target.value)}
+                className="form-control"
+                required
+              />
+            </div>
+            <div className={`form-group ${styles.formGroupCustom}`}>
+              <label
+                className={
+                  password
+                    ? `${styles.floatLabel}`
+                    : `${styles.noLabel}`
+                }
+              >
+                {' '}
+                Password:
+              </label>
+              <input
+                type="password"
+                value={password}
+                placeholder="Password"
+                onChange={e => setPassword(e.target.value)}
+                className="form-control"
+                required
+              />
+            </div>
+            <div className={`form-group ${styles.formGroupCustom}`}>
+              <label
+                className={
+                  confirmPassword
+                    ? `${styles.floatLabel}`
+                    : `${styles.noLabel}`
+                }
+              >
+                Confirm Password:
```

Comment:
Let's remove confirm password. It'll be less things for the user to enter and confirming password is becoming more and more rare.

Reply:
okay i'll remove that and push now